services/nginz/Dockerfile

```diff
@@ -0,0 +1,162 @@
+# Requires docker >= 17.05 (requires support for multi-stage builds)
+FROM alpine:3.6 as libzauth-builder
+
+# Compile libzauth
+COPY libs/libzauth /src/libzauth
+RUN cd /src/libzauth/libzauth-c \
+    && export PREFIX=/usr \
+    && apk add --no-cache make bash cargo libsodium-dev \
+    && make install
+
+# Nginz container
+FROM alpine:3.6
+
+ENV NGINX_VERSION 1.12.1
+
+# Install libzauth
+COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
+COPY --from=libzauth-builder /usr/lib/libzauth.so /usr/lib/libzauth.so
+COPY --from=libzauth-builder /usr/lib/pkgconfig/libzauth.pc /usr/lib/pkgconfig/libzauth.pc
+
+COPY services/nginz/third_party /src/third_party
+
+# Install nginz (nginx including the zauth module)
+# (taken mostly from https://github.com/nginxinc/docker-nginx/blob/master/stable/alpine/Dockerfile)
+RUN GPG_KEYS=B0F4253373F8F6F510D42178520A9993A1C052F8 \
+    && CONFIG="\
+        --prefix=/etc/nginx \
+        --sbin-path=/usr/sbin/nginx \
+        --modules-path=/usr/lib/nginx/modules \
+        --conf-path=/etc/nginx/nginx.conf \
+        --error-log-path=/var/log/nginx/error.log \
+        --http-log-path=/var/log/nginx/access.log \
+        --pid-path=/var/run/nginx.pid \
+        --lock-path=/var/run/nginx.lock \
+        --http-client-body-temp-path=/var/cache/nginx/client_temp \
+        --http-proxy-temp-path=/var/cache/nginx/proxy_temp \
+        --http-fastcgi-temp-path=/var/cache/nginx/fastcgi_temp \
+        --http-uwsgi-temp-path=/var/cache/nginx/uwsgi_temp \
+        --http-scgi-temp-path=/var/cache/nginx/scgi_temp \
+        --user=nginx \
+        --group=nginx \
+        --with-http_ssl_module \
```

Comment:
Perhaps include fewer modules like we do [here](https://github.com/wireapp/wire-server/blob/develop/services/nginz/Makefile#L28-L33) ?

Reply:
True, fixed in https://github.com/wireapp/wire-server/pull/69/commits/cf47931bc29d5a51b76cbf36620776b6257bf20a